test-result-summary-client/src/TestCompare/TestCompare.jsx

```diff
@@ -16,7 +16,9 @@ export default class TestCompare extends Component {
         }, {
             url: "https://ci.eclipse.org/openj9", buildName: "Test-Sanity-JDK10-aix_ppc-64_cmprssptrs", buildNum: 35, testName: "jit_jitt_3"
         }],
-        tests: [{}, {}]
+        tests: [{}, {}],
+        timeStamp: false,
+        deepSmith: false

```

Comment:
Please prefix the boolean variable with a verb to avoid confusion. For example, `removeTimestamp` and `applyDeepSmithMatch`